kubernetes/akka-cluster.yml

```diff
@@ -19,14 +19,22 @@ spec:
       containers:
         - name: appka
           image: bootstrap-demo-kubernetes-api:1.3.3.7
```

Comment:
I kept this version number, but when writing the guide we should probably use dynamic version as https://developer.lightbend.com/guides/openshift-deployment/lagom/building-using-maven.html#git-hash-based-version-numbers